lib/PuppeteerSharp/Page.cs

```diff
@@ -761,8 +763,37 @@ private void OnDialog(PageJavascriptDialogOpeningResponse message)
             Dialog?.Invoke(this, new DialogEventArgs(dialog));
         }
 
-        private void OnConsoleAPI(MessageEventArgs e)
+        private async Task OnConsoleAPI(PageConsoleResponse message)
         {
+            // TODO Do we need to translate this part ?
+//            if (event.type === 'debug' && event.args.length && event.args[0].value === 'driver:page-binding') {
+//                const {name, seq, args} = JSON.parse(event.args[1].value);
+//                const result = await this._pageBindings[name](...args);
+//                const expression = helper.evaluationString(deliverResult, name, seq, result);
+//                this._client.send('Runtime.evaluate', { expression, contextId: event.executionContextId }).catch(debugError);
+//
+//                function deliverResult(name, seq, result) {
+//                    window[name]['callbacks'].get(seq)(result);
+//                    window[name]['callbacks'].delete(seq);
+//                }
+//                return;
+//            }
+            
+            if (Console?.GetInvocationList().Length == 0) {
+                foreach (var arg in message.Args)
+                {
+                    await Helper.ReleaseObject(_client, arg);
+                }
+                
+                return;
+            }
+            
+            var handles = message.Args
+                .Select(_ => (JSHandle)_frameManager.CreateJsHandle(message.ExecutionContextId, _))
```

Comment:
do we need to cast here?

Reply:
Yes, I don't really understand why, but it doesn't recognize the return type of CreateJsHandle : 

    Unable to cast object of type 'System.Collections.Generic.List<dynamic>' to type 'System.Collections.Generic.IList<PuppeteerSharp.JSHandle>'

Reply:
it's because `message.Args` is dynamic

Reply:
Yes,  that's normal, it's the original data, not yet transformed into a JSHandle

Reply:
too bad.
ok

Reply:
First time I see `_` in use, fancy!

What would be the benefit of using `_` instead of defaulting that argument to null in the method?